src/3rdparty/walkontable/css/walkontable.css

```diff
@@ -209,6 +209,12 @@
   overflow: auto;
 }
 
+.handsontable .ht_master thead,
+.handsontable .ht_master tr th,
+.handsontable .ht_clone_left thead {
+  visibility: hidden;
+}
```

Comment:
@nikersify the linter failed in the build run https://github.com/handsontable/handsontable/runs/1473045254, because of the outdated GH action `actions/setup-node`. Can you please cherry pick a fix from `develop` that updated the GH workflow recently?

Reply:
https://github.com/handsontable/handsontable/pull/6157/commits/0d5e75d6049c31c26d0fdbee364a9c681b73c75a